bot/exts/halloween/candy_collection.py

```diff
@@ -197,20 +185,15 @@ async def candy(self, ctx: commands.Context) -> None:
             '\N{SPORTS MEDAL}'
         )
 
-        top_sorted = sorted(self.candy_json['records'], key=lambda k: k.get('record', 0), reverse=True)
+        top_sorted = sorted(((user_id, score) for user_id, score in self.candy_records.items()),
+                            key=lambda x: x[1], reverse=True)
```

Comment:
this can be formatted in a  better way.
```suggestion
        top_sorted = sorted(
            ((user_id, score) for user_id, score in self.candy_records.items()),
            key=lambda x: x[1],
            reverse=True
        )
```